packages/core/src/index.js.flow

```diff
@@ -77,10 +77,13 @@ declare export function merge <A> (s1: Stream<A>, s2: Stream<A>): Stream<A>
 declare export function merge <A> (s1: Stream<A>): (s2: Stream<A>) => Stream<A>
 declare export function mergeArray <A> (ss: Array<Stream<A>>): Stream<A>
 
-declare export function sample <A, B, C> (f: (A, B) => C, sampler: Stream<A>, sampled: Stream<B>): Stream<C>
-declare export function sample <A, B, C> (f: (A, B) => C): (sampler: Stream<A>, sampled: Stream<B>) => Stream<C>
-declare export function sample <A, B, C> (f: (A, B) => C, sampler: Stream<A>): (sampled: Stream<B>) => Stream<C>
-declare export function sample <A, B, C> (f: (A, B) => C): (sampler: Stream<A>) => (sampled: Stream<B>) => Stream<C>
+declare export function withLatestFrom <A, B> (values: Stream<A>, stream: Stream<B>): Stream<A>
```

Comment:
I wouldn't bother with the second type parameter.
`withLatestFrom <A> (values: Stream<A>, stream: Stream<any>): Stream<A>`

Reply:
Yeah, seems reasonable in this case :+1: ... I've just gotten in the habit of avoiding `any`

Reply:
Maybe better `*`?

Reply:
Hmmm, I don't see any advantage to `*` over `any` in this particular case.  The type is completely ignored.  Do you see any advantage _in this particular case_?

Reply:
Just not using `any` :)